includes/embeds/class-amp-youtube-embed-handler.php

```diff
@@ -124,17 +124,49 @@ private function parse_props( $html, $url, $video_id ) {
 		return $props;
 	}
 
+	/**
+	 * Gets AMP-compliant markup for the YouTube shortcode.
+	 *
+	 * @deprecated 1.5.0 This was moved to Jetpack, in jetpack_amp_youtube_shortcode().
+	 *
+	 * @param array $attr The YouTube attributes.
+	 * @return string YouTube shortcode markup.
+	 */
+	public function shortcode( $attr ) {
+		_deprecated_function( __METHOD__, '1.5.0' );
+		$url = false;
+
+		if ( isset( $attr[0] ) ) {
+			$url = ltrim( $attr[0], '=' );
+		} elseif ( function_exists( 'shortcode_new_to_old_params' ) ) {
+			$url = shortcode_new_to_old_params( $attr );
+		}
+
+		if ( empty( $url ) ) {
+			return '';
+		}
+
+		$video_id = $this->get_video_id_from_url( $url );
+
+		return $this->render( compact( 'video_id' ), $url );
+	}
+
 	/**
 	 * Render oEmbed.
 	 *
 	 * @see \WP_Embed::shortcode()
 	 * @deprecated This is no longer being used.
 	 *
+	 * @param array  $matches URL pattern matches.
+	 * @param array  $attr    Shortcode attribues.
+	 * @param string $url     URL.
 	 * @return string Rendered oEmbed.
 	 */
-	public function oembed() {
+	public function oembed( $matches, $attr, $url ) {
+		unset( $matches, $attr );
 		_deprecated_function( __METHOD__, '1.5.0' );
-		return '';
+
+		return $this->shortcode( [ $url ] );
 	}
```

Comment:
Since these methods aren't used anymore, can't they just be made to return an empty string while having the `_deprecated_function()` calls? 

We might as well just remove them altogether because they aren't being used.

Reply:
Good point, https://github.com/ampproject/amp-wp/pull/3678/commits/4726922a1788e0d034e6bf5c955051c653d0231b removes `AMP_YouTube_Embed_Handler::shortcode()` and `oembed()`, as they're not used anywhere.